synapse/storage/events.py

```diff
@@ -1548,6 +1557,97 @@ def _store_redaction(self, txn, event):
             (event.event_id, event.redacts),
         )
 
+    @defer.inlineCallbacks
+    def _censor_redactions(self):
+        """Censors all redactions older than a month that haven't been censored.
```

Comment:
> older than a month

Should be changed to better reflect the current state of this PR.

Reply:
:man_facepalming: I'm terrible at comments